pdm/models/pip_shims.py

```diff
@@ -83,13 +79,6 @@ def get_flag(var, fallback, expected=True, warn=True):
             "WITH_PYMALLOC", is_cpython, warn=False
         ):
             m = "m"
-        if python_version < (3, 3) and get_flag(
-            "Py_UNICODE_SIZE",
-            sys.maxunicode == 0x10FFFF,
-            expected=4,
-            warn=False,
-        ):
-            u = "u"
```

Comment:
I haven't got a single clue what this does, but it's unreachable

Reply:
It’s for marking whether `u""` literals are supported (added in Python 3.3), so yeah irrelevant now.